wee_slack.py

```diff
@@ -2238,8 +2238,7 @@ def scrolled_cb(signal, sig_type, data):
             if not w.config_get_plugin('switch_buffer_on_join'):
                 w.config_set_plugin('switch_buffer_on_join', "1")
 
-            if w.config_get_plugin('channels_not_on_current_server_color'):
-                w.config_option_unset('channels_not_on_current_server_color')
+            w.config_option_unset('channels_not_on_current_server_color')
```

Comment:
Why this bit? It's used for cleaning up an obsolete setting and should not be removed unless it causes problems (and then in a separate PR).